AbletonMCP_Remote_Script/handlers/browser.py

```
@@ -0,0 +1,653 @@
+                result_item = {
+                    "name": item.name,
+                    "is_folder": (hasattr(item, "is_folder") and item.is_folder) or (hasattr(item, "children") and bool(item.children)),
+                    "is_device": hasattr(item, "is_device") and item.is_device,
+                    "is_loadable": hasattr(item, "is_loadable") and item.is_loadable,
+                    "uri": item.uri if hasattr(item, "uri") else None,
+                }
+                results.append(result_item)
+            if hasattr(item, "children"):
+                try:
+                    children = item.children
+                except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)